pkgs/development/python-modules/readabilipy/default.nix

```
@@ -34,23 +53,36 @@ buildPythonPackage rec {
     regex
   ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  postPatch = ''
+    ln -s ${javascript}/lib/node_modules/ReadabiliPy/node_modules readabilipy/javascript/node_modules
```

```suggestion
    ln -s $javascript/lib/node_modules/ReadabiliPy/node_modules readabilipy/javascript/node_modules
```

This makes `javascript` overrideable

---

Should I just refactor to avoid `rec` entire?